app/models/conversation.rb

```diff
@@ -24,7 +24,7 @@ def max_participants
   def local_recipients
     recipients.each do |recipient|
       if recipient.local?
-        if recipient.owner.contacts.where(:person_id => self.author.id).count == 0
+        if (recipient.owner.contacts.where(person_id: author.id).count == 0) && (!author.owner.podmin_account?)
```

Comment:
Actually the parens seem unnecessary here, we could also get rid of the negation here: `unless recipient.owner.contacts.where(person_id: author.id).any? || author.owner.podmin_account?`

Thinking about this, are we sure the author is local? I think it's not guaranteed that we have an owner.